apps/kitchensink-react/src/DocumentCollection/DocumentEditorRoute.tsx

```diff
@@ -125,6 +119,32 @@ function Editor() {
   )
 }
 
+function Editor() {
+  const {data: documents} = useDocuments({
+    documentType: 'author',
+    batchSize: 1,
+  })
+
+  const [docHandle, setDocHandle] = useState<DocumentHandle<'author'> | null>(null)
+
+  useEffect(() => {
+    if (documents.length > 0) {
+      const doc = documents[0]
+      setDocHandle(doc)
+    }
+  }, [documents])
```

Comment:
Just wondering why `useState` and `useEffect` are used?

Reply:
none reasons except old habits -- fixed!